chatterbox/src/Main/Header/index.js

```diff
@@ -0,0 +1,132 @@
+import { withStyles } from 'material-ui/styles';
+import './index.css';
+import Oval from '../../assets/Oval.png';
+import Oval_2 from '../../assets/Oval_2.png';
+import AppBar from 'material-ui/AppBar';
+import Toolbar from 'material-ui/Toolbar';
+import IconButton from 'material-ui/IconButton';
+import MenuIcon from 'material-ui-icons/Menu';
+import Menu, { MenuItem } from 'material-ui/Menu';
+import Button from 'material-ui/Button';
+
+const styles = theme => ({
+    root: {
+        height: ' 103px',
+        width: '1440px'
+    },
+
+    flex: {
+        flex: 1
+    },
+    menuButton: {
+        marginLeft: -12,
+        marginRight: 20
+    },
+
+    images: {
+        borderRadius: 50
+    }
+});
+
+class MenuAppBar extends React.Component {
+    state = {
+        auth: true,
+        anchorEl: null
+    };
+
+    handleChange = (event, checked) => {
+        this.setState({ auth: checked });
+    };
+
+    handleMenu = event => {
+        this.setState({ anchorEl: event.currentTarget });
+    };
+
+    handleClose = () => {
+        this.setState({ anchorEl: null });
+    };
+
+    settings() {
+        this.props.history.push('/settings');
+    }
+
+    render() {
+        const { classes } = this.props;
+        const { auth, anchorEl } = this.state;
+        const open = Boolean(anchorEl);
+
+        return (
+            <div className={classes.root} color="#01B9BD">
+                <AppBar position="static" className="AppBarymai">
+                    <Toolbar>
+                        <div className={classes.flex}>
+                            <div className="appbar-buttons">
+                                <Button raised className={classes.button}>
+                                    New Chat
+                                </Button>
+                                &nbsp;&nbsp;
+                                <Button raised className={classes.button}>
+                                    New Group
+                                </Button>
+                            </div>
+                        </div>
+
+                        {auth && (
+                            <div className="appbar-buttons-right">
+                                <Button
+                                    aria-owns={open ? 'menu-appbar' : null}
+                                    aria-haspopup="true"
+                                    onClick={this.handleMenu}
+                                >
+                                    <h6 id="Addie_div">Addie Hogan </h6> &nbsp;
+                                    <img
+                                        className={classes.images}
+                                        height="60"
+                                        src={Oval}
+                                        alt=" "
+                                    />
+                                </Button>
+
+                                <Button aria-owns={open ? 'menu-appbar' : null} color="contrast">
```

Comment:
Can't this (open ? 'menu-appbar' : null) be replaced by with: open && 'menu-appbar'

Reply:
added the required change. replaced (open ? 'menu-appbar' : null) by open && 'menu-appbar'